stix_shifter/stix_translation/src/modules/azure_sentinel/README.md

```diff
@@ -0,0 +1,375 @@
+# Azure Sentinel - ISC UDS Connector
```

Comment:
Remove reference to ISC UDS.

Reply:
Removed ISC UDS reference from README.md